lib/src/widgets/pretty_qr_error.dart

```diff
@@ -0,0 +1,56 @@
+// ignore_for_file: no-object-declaration
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/foundation.dart';
+import 'dart:math' as math;
+
+class PrettyQrErrorWidget extends StatelessWidget {
```

Comment:
I think it's better to move this widget in the same library, but make it private?